bunq/sdk/model/generated/endpoint.py

```diff
@@ -1,14 +1,14 @@
-# -*- coding: utf-8 -*-
-from bunq.sdk import client
-from bunq.sdk import context
-from bunq.sdk import exception
-from bunq.sdk import security
+from bunq.sdk.exception.bunq_exception import BunqException
+from bunq.sdk.http.api_client import ApiClient
+from bunq.sdk.http.bunq_response import BunqResponse
 from bunq.sdk.json import converter
-from bunq.sdk.model import core
+from bunq.sdk.model.core.anchored_object_interface import AnchoredObjectInterface
 from bunq.sdk.model.generated import object_
+from bunq.sdk.security import security
+from bunq.sdk.model.core.bunq_model import BunqModel
 
 
-class BillingContractSubscription(core.BunqModel):
+class BillingContractSubscription(BunqModel):
```

Comment:
why did you factor out all other classes into their own files and left these in one big file?

Reply:
The two classes in the `generated` package are generated by the SDK Generator. Fixing up the SDK Generator to not put everything into single files is another big job for the future...